agent-identity/python/on-behalf-of-flow/crew-ai/main.py

```
@@ -0,0 +1,140 @@
+        base_url=os.getenv("ASGARDEO_BASE_URL"),
+        client_id=os.getenv("CLIENT_ID"),
+        redirect_uri=os.getenv("REDIRECT_URI")
+    )
+    AGENT_CONFIG = AgentConfig(
+        agent_id=os.getenv("AGENT_ID"),
+        agent_secret=os.getenv("AGENT_SECRET")
+    )
+
+    async with AgentAuthManager(ASGARDEO_CONFIG, AGENT_CONFIG) as auth_manager:
+        agent_token = await auth_manager.get_agent_token(["openid", "email"])
+        auth_url, state, code_verifier = auth_manager.get_authorization_url_with_pkce(["openid", "email"])
```

Ruff (0.15.1)

[warning] Unpacked variable `state` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)